html5lib/tests/test_sanitizer.py

```diff
@@ -63,25 +63,32 @@ def test_sanitizer():
     for ns, tag_name in sanitizer.allowed_elements:
         if ns != constants.namespaces["html"]:
             continue
-        if tag_name in ['caption', 'col', 'colgroup', 'optgroup', 'option', 'table', 'tbody', 'td',
-                        'tfoot', 'th', 'thead', 'tr', 'select']:
+        if tag_name in ['caption', 'col', 'colgroup', 'optgroup', 'option',
```

Comment:
Can we avoid unnecessary code-formatting changes in this PR? It makes it less obvious what has actually changed in this PR.